### ISSUE TITLE
Add ability to rename schedules

### ISSUE THREAD

**Issue by ryan-conn** [aggie-coding-club/Rev-Registration#318]
After #317 is done, we need to actually implement the design we come up with, this will involve changing the redux for schedules, adding additional components, and creating tests for it

**Comment by gannonprudhomme** [comment]
Closing, duplicate of #296 . Re-open if this is wrong though
